File: sdks/csharp/HookSniff/OperationalWebhook.cs
using System.Collections.Generic;
using System.Threading.Tasks;
namespace HookSniff {
    public class OperationalWebhook {
        private readonly HookSniffClient _client;
        public OperationalWebhook(HookSniffClient client) { _client = client; }
        public Task<List<Dictionary<string, object>>> List() => _client.HookSniffHttpClient.GetAsync<List<Dictionary<string, object>>>("/api/v1/operational-webhooks");
        public Task<Dictionary<string, object>> Create(Dictionary<string, object> body) => _client.HookSniffHttpClient.PostAsync<Dictionary<string, object>>("/api/v1/operational-webhooks", body);
        public Task<Dictionary<string, object>> Get(string id) => _client.HookSniffHttpClient.GetAsync<Dictionary<string, object>>($"/api/v1/operational-webhooks/{id}");
        public Task<Dictionary<string, object>> Update(string id, Dictionary<string, object> body) => _client.HookSniffHttpClient.PutAsync<Dictionary<string, object>>($"/api/v1/operational-webhooks/{id}", body);
        public Task Delete(string id) => _client.HookSniffHttpClient.DeleteAsync($"/api/v1/operational-webhooks/{id}");
        public Task<List<Dictionary<string, object>>> ListDeliveries(string id) => _client.HookSniffHttpClient.GetAsync<List<Dictionary<string, object>>>($"/api/v1/operational-webhooks/{id}/deliveries");
    }
}

File: sdks/ruby/lib/hooksniff/operational_webhook.rb
# frozen_string_literal: true
module HookSniff
  class OperationalWebhook
    def initialize(api_client) = @api_client = api_client
    def list = @api_client.execute(method: :get, path: "/api/v1/operational-webhooks")
    def create(body) = @api_client.execute(method: :post, path: "/api/v1/operational-webhooks", body: body)
    def get(id) = @api_client.execute(method: :get, path: "/api/v1/operational-webhooks/#{id}")
    def update(id, body) = @api_client.execute(method: :put, path: "/api/v1/operational-webhooks/#{id}", body: body)
    def delete(id) = @api_client.execute(method: :delete, path: "/api/v1/operational-webhooks/#{id}")
    def list_deliveries(id) = @api_client.execute(method: :get, path: "/api/v1/operational-webhooks/#{id}/deliveries")
  end
end
